apps/dialtone-documentation/docs/components/input.md

```diff
@@ -299,6 +299,73 @@ vueCode='
 '
 showHtmlWarning />
 
+### With left icon and clear button
```

Comment:
I guess we could name it just `Search Input`

Reply:
ha I was writing this exact message but didn't post it. Yeah I agree, users will come here looking specifically for search input since it's a common case.